azure-pipelines/nightly-snapshot-releases/Latest Common VSTS Release.yml

```diff
@@ -1,15 +1,14 @@
-# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
-#       env:
-#           MY_ACCESS_TOKEN: $(System.AccessToken)
-# Variable 'android:serverUrl' was defined in the Variables tab
-# Variable 'ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME' was defined in the Variables tab
-# Variable 'test_repo_branch' was defined in the Variables tab
-# Variable 'test_repo_dir' was defined in the Variables tab
-# Variable Group 'Android Common Variables' was defined in the Variables tab
-# Cron Schedules have been converted using UTC Time Zone and may need to be updated for your location
+# File: azure-pipelines\nightly-snapshot-releases\Latest Common VSTS Release.yml
+# Description: Assemble & publish Latest common release nightly to internal feed
+name: $(date:yyyyMMdd)$(rev:.r)
```

Comment:
I tried to do it but once the name is defined I cannot change it

Reply:
can we read the version number from build.gradle and set it later. It will be nice to see which version of common did the pipeline published.

You can open a task for this (Naming the piepline runs to reflect lib versions)